tests/puppeteer/tools/urls.js

```diff
@@ -0,0 +1,183 @@
+module.exports = [
+  /* {
```

Comment:
Remove useless comment

Reply:
Done, forget i did that to fasten my tests